src/cmdlib.sh

```diff
@@ -520,7 +536,11 @@ impl_rpmostree_compose() {
         fi
         # And remove the old one
         rm -vf "${workdir}"/cache/cache.qcow2
-        compose_qemu_args+=("-drive" "if=none,id=cache,discard=unmap,file=${workdir}/cache/cache2.qcow2" \
+        local cachedriveargs="discard=unmap"
```

Comment:
I think the answer is under the `qcow2` section:

> pass-discard-request
> Whether discard requests to the qcow2 device should be forwarded to the data source (on/off; default: on if discard=unmap is specified, off otherwise)